sources/packages/backend/apps/load-test-gateway/src/auth/load-test-user-token.ts

```diff
@@ -0,0 +1,5 @@
+import { LoadTestAuthorizedParties } from "./authorized-parties.enum";
+
+export interface LoadTestUserToken {
```

Comment:
The authorized parties on the SIMS-Api were introduced because the clients would be authentication using different providers, which will not be the case. 
I may be missing something but I would recommend not including this verification for this API.

Reply:
I get it. As we have token validation at audience level, AZP may not be significant here.